lib/common.js

```diff
@@ -170,14 +170,14 @@ const getIdxLookups = (fields, lookups, version = CURRENT_DD_VERSION) => {
 
 const getLastPart = (str, char) => str.substr(str.lastIndexOf(char) + 1);
 
-const getAdvertisedCountPerResourcesByType = ({ fields, lookups, version }) => {
+const getAdvertisedCountPerResourcesByType = ({ fields, lookups, version = CURRENT_DD_VERSION }) => {
   const reducedFields = fields.reduce(function (r, a) {
     r[a.resourceName] = r[a.resourceName] || [];
     r[a.resourceName].push(a);
     return r;
   }, Object.create(null));
   const advertisedCount = {};
-  const idxLookups = getIdxLookups(fields, lookups, version);
+  const idxLookups = getIdxLookups(fields, lookups, version ?? CURRENT_DD_VERSION);
```

Comment:
Rather than using a fallback for version here, it should be done in the `getIdxLookups` method.